torchdynamo/variables/tensor.py

```diff
@@ -311,7 +311,10 @@ def call_isinstance(self, tensor_type):
 
         def check_type(ty):
             if ty not in tensortype_to_dtype:
-                return self.python_type() is ty
+                if self.python_type() is torch.nn.Parameter and ty is torch.Tensor:
```

Comment:
use `isinstance(self.python_type(), ty)`

Reply:
sync up with @anijain2305 offline and find ```issubclass``` is the most concise way.